src/components/Navbar/components/CurrentLocationWidget/CurrentLocationWidget.tsx

```diff
@@ -0,0 +1,68 @@
+import { weatherSuccessResponse } from 'mocks/mockData';
+import ContentLoader from 'react-content-loader';
+import { Weather } from 'models/Weather';
+import { Container, WeatherIcon, Description, DegreeSign } from './CurrentLocationWidget.styled';
+
+const CurrentLocationWidget = () => {
+  let isLoading = false;
+  const data: Weather | any = weatherSuccessResponse; // temporary solution in order not to use
+
+  const renderContent = () => {
```

Comment:
**question** do we really need this render function?

Reply:
Well, we don't need it, but I decided to follow the convention from previous features